src/OSPSuite.Core/Converters/v12/Converter110To120.cs

```diff
@@ -186,6 +187,15 @@ public void Visit(SimulationConfiguration simulationConfiguration)
          simulationConfiguration.ModuleConfigurations.Each(Visit);
       }
 
+      public void Visit(ParameterValuesBuildingBlock buildingBlock)
+      {
+         buildingBlock.Each(x =>
+         {
+            if (x.Formula != null)
```

Comment:
I would add a comment here explaing this code .Eg 
"in v11 and prior, both values were saved to pkml bla bla"